### PR TITLE
add configuration for maximum queued request count

### PR DESCRIPTION
**Pull request by karok2m**
## Purpose

Added CLI option to control:

- maximum queued request count

Similar to #274.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Review by cristipogacean — APPROVED** [review]

**Review by dacolgit — APPROVED** [review]
:shipit:

File: src/OpcApplicationConfiguration.cs [hunk 1]
```diff
@@ -240,8 +242,7 @@ private static void ConfigureUserTokenPolicies(IApplicationConfigurationBuilderS
 
     private const int MAX_MESSAGE_QUEUE_SIZE = 200000;
     private const int MAX_NOTIFICATIONS_PER_PUBLISH = 200000;
-    private const int MAX_SUBSCRIPTION_COUNT = 200;
-    private const int MAX_PUBLISH_REQUEST_COUNT = MAX_SUBSCRIPTION_COUNT;
+    private const int MAX_PUBLISH_REQUEST_COUNT = 200;
```
Comment (luiscantero) [inline src/OpcApplicationConfiguration.cs:245]: >200 [](http://example.com/codeflow?start=50&length=3)

The readme says 2000 and here says 200. Which is correct?

Comment (karok2m) [reply src/OpcApplicationConfiguration.cs:245]: The change and readme is about queued request count. That is different from publish request count.

**Review by luiscantero — APPROVED** [review]
:shipit:

**Pull request merged**